### ISSUE TITLE
Indentation lint is over-eager w/ multiple @import files

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#442]
This one is kind of weak, but here me out :smile: 

a.scss

```scss
@import 'foo',
        'bar';

@import 'baz';

 @import 'xxx', 'yyy';

a {
  @include transition($property: opacity,
                      $timing: ease-in-out);
  background: inline-image('crazy-url.svg')
      center no-repeat;
}
```

```bash
$ bundle exec bin/scss-lint a.scss
a.scss:2 [W] Indentation: Line should be indented 0 spaces, but was indented 8 spaces
a.scss:6 [W] Indentation: Line should be indented 0 spaces, but was indented 1 spaces
a.scss:6 [W] Indentation: Line should be indented 0 spaces, but was indented 1 spaces
```

So what I would _not_ expect is: 1) the first lint, because a comma-separated list _feels_ like its all one node, starting at line 1, offset 0, and 2) the 3rd lint, for the same reason (here Indentation is double-reporting). My "comma-separated list is all one node" heuristic comes from the non-lints in the example, like the `@include` and `background` lines.

I have a fix for this, but it's unfortunately a little hacky: it depends on `previous_node()` which depends on a Node's `==`, which is ill-defined for `ImportNode`. So I have a monkey patch for that. The basic reason for the tricky/hacky fix is that the comma-syntax is just a shortcut; Sass creates a new `ImportNode` for each, and none of them contain the `@import` source.

**Comment by sds** [comment]
Can you elaborate on why `ImportNode#==` is ill-defined? Can we not just compare the current node to its previous node and if they have the same `line` not report the lint? (I haven't looked into this deeply, but a monkey patch seems unnecessary to me)

**Comment by srawlins** [comment]
It's that `#==` is not defined on `ImportNode`. It's defined on [`Node`](https://github.com/sass/sass/blob/stable/lib/sass/tree/node.rb#L141):

```ruby
def ==(other)
  self.class == other.class && other.children == children
end
```

so ImportNodes are always considered equal :( so the `previous_node()` calculation comes out wrong.

**Comment by sds** [comment]
That's unfortunate. Alright, I'd be open to merging a fix even if it requires a monkey patch for `ImportNode#==`. Thanks!
